documentation/1.10/content/userguide/tools/create-aux-image.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Create Auxiliary Image"
+date: 2019-02-23
+draft: false
+weight: 2
+description: "The createAuxImage command creates a new container image with WDT models, archives, and variable files."
+---
+
+
+The `createAuxImage` command helps build a container image from a given base OS image. 
+The required option for the command is marked. There are a number of optional parameters for the create feature.
```

Comment:
Just as a suggestion, I would reverse the order of these two sentences: There are a number of optional parameters for the create feature. The required option for the command is marked.